### PR TITLE
The plush toy now has an expanded vocabulary!

### PR DESCRIPTION
**Pull request by wiverns**
## About the PR 
Expands the Plush Toy's vocabulary by separating it from the ouija board list! It now pulls from things players say more often, and adds a bunch of new words and phrases, such as:
- Oderint dum metuant, "Let them hate so long as they fear"
- Reactions (Uh oh, woopsie, ouch!)
- A few curse words
- ...and more!

## Why's this needed? 
More variety = more fun!

## Changelog

```changelog
(u)Wivernshy
(+)Plush toy vocabulary has been improved!
```

### CODE REVIEW

File: code/procs/gamehelpers.dm [hunk 1]
```diff
@@ -776,3 +776,81 @@ proc/get_ouija_word_list(var/atom/movable/source = null, var/words_min = 5, var/
...
+/proc/get_initial_item_health(obj/item/I)
+	if (initial(I.health))
+		return initial(I.health)
+	else
+		var/weight_class = initial(I.w_class)
+		switch (weight_class)
+			if (W_CLASS_TINY to W_CLASS_NORMAL)
+				return weight_class + 1
+			else
+				return weight_class + 2
```
Comment (TobleroneSwordfish) [inline code/procs/gamehelpers.dm:791]: ?

Comment (TobleroneSwordfish) [reply code/procs/gamehelpers.dm:791]: Oh I see what you've done, probably best to revert to your first commit and then merge again and select "accept both"

Comment (wiverns) [reply code/procs/gamehelpers.dm]: reverted, not sure if I did what you said to do correctly though?

Comment (TobleroneSwordfish) [reply code/procs/gamehelpers.dm]: yep, now you need to merge again and keep both upstream and local changes

File: code/procs/gamehelpers.dm [hunk 2]
```diff
@@ -776,3 +776,84 @@ proc/get_ouija_word_list(var/atom/movable/source = null, var/words_min = 5, var/
...
+>>>>>>> master
```
Comment (TobleroneSwordfish) [inline code/procs/gamehelpers.dm:859]: These bits are git merge markers, they're meant to be removed when you accept changes. Probably best to look up a guide on how to resolve git merge conflicts or feel free to ask in the goonstation discord

**Comment by pali6** [comment]
Apart from the merge conflict markers this also doesn't need a major changelong entry. Also if the code is copypasted it should probably be refactored not to be copypasted.

**Review by pali6 — APPROVED** [review]
I fixed the bad merges and reduced copypaste, hope you don't mind.

**Pull request merged**